doc/devel/coding_guide.rst

```diff
@@ -7,6 +7,8 @@
    .checklist li p {display: inline; }
    </style>
 
+.. highlight:: bash
```

Comment:
I'd be inclined to use  `.. code-block:: bash` instead in the 3 use cases below.

While we don't have any other code now, the default highlighting is python and if somebody adds a python snippet here later, we probably won't notice that it's highlighted as bash.

But I'll leave the decision to you.

Reply:
sure, fixed.